backend/src/app.js

```diff
@@ -58,11 +58,28 @@ router.get('/v1/templates/:name', (ctx) => {
         ctx.body = e;
         return;
     }
+
+    let readme = '';
+    const readmeTemplatePath = path.join(ctx.templatePath, ctx.params.name, 'README.md');
+    if (fs.existsSync(readmeTemplatePath)) {
+        readme = getReadme(readmeTemplatePath);
+    }
+
     ctx.body = {
+        readme,
```

Comment:
@Thebigbignooby What do you think if we prevent sending readme when it's not defined? This will allow us to render markdown component only when readme is present